src/curd.js

```diff
@@ -54,8 +56,22 @@ export const populateList = () => {
       divSmall.classList.add('divSmall');
       const checkbox = document.createElement('input');
       checkbox.setAttribute('type', 'checkbox');
+
+      checkbox.setAttribute('data-index', x);
+
       checkbox.setAttribute('id', `chk${x}`);
 
+      // add event listener to the check box change
+      checkbox.addEventListener('change', (event) => {
+        // Retrieve the value of 'data-index' attribute
+        const dataIndex = event.target.getAttribute('data-index');
+
+        // update item's object's value  for completed  key upon user actions
+        // Do something with the dataIndex value
+        updateStatus(todoList, saveTodoListToLocalStorage, dataIndex);
+        console.log(`Checkbox with data-index ${dataIndex} was changed.`);
```

Comment:
- [ ] @umarahgul It's not good practice to have console.log statements in our codebase, kindly consider removing this console.log statement.